### ISSUE TITLE
Switch to poetry for build/install

### ISSUE THREAD

**Issue by mjpost** [mjpost/sacrebleu#202]

**Comment by mjpost** [comment]
Anyone (@BrightXiaoHan @martinpopel @ozancaglayan) have any experience with this (and time this week to implement it or at least provide a draft framework)?

**Comment by martinpopel** [comment]
no and no :-(

**Comment by ozancaglayan** [comment]
I played with it for another project of mine, couple of weeks ago. For the complexity of that project, poetry had failed so I switched to `pipenv`. Unfortunately, I don't have time this week for this. What's the motive for this? AFAIK Pipenv/poetry etc, are useful for reproducible build/dev environments rather than installable artefacts pushed to PyPI. They're good for pinning package versions to exact semantic versions, which is something that can't be achieved for end-user installation purposes such as sacreBLEU, as hard-pinning is likely to break other packages installed to the same environment.
